browser-test/src/northstar_link_tags.test.ts

```diff
@@ -0,0 +1,17 @@
+import {test, expect} from './support/civiform_fixtures'
+import {enableFeatureFlag} from './support'
+
+test.describe('navigating to a deep link', {tag: ['@northstar']}, () => {
+  test.beforeEach(async ({page}) => {
+    await enableFeatureFlag(page, 'north_star_applicant_ui')
+  })
+
+  test('has favicon link', async ({page, applicantQuestions}) => {
+    await applicantQuestions.gotoApplicantHomePage()
+
+    const linkTagLocator = page.locator('link[rel="icon"]')
+
+    await expect(linkTagLocator).toHaveAttribute('href')
+    expect(await linkTagLocator.getAttribute('href')).not.toBeNull()
```

Comment:
(blocking) If the href is an empty string this will pass. `toHaveAttribute` has an overload that takes a regex. Recommend having a regex check that something is there. [See docs](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-attribute-option-value).

Also unless there's no other option you want to avoid using Playwright's older assertion style in favor of their more modern and recommended approach with web first / auto retrying assertions. [See docs](https://playwright.dev/docs/test-assertions).